### DISCUSSION TITLE
Add support for telegram chat folders

### DISCUSSION THREAD

**Discussion by vinniec2** [d99kris/nchat#234] [Ideas]
Today I tried to compile chat, it was a long task (underpowered pc) however it was worth it, nchat compared to alternatives is really light and fast.
So far what I have noticed would be useful within minutes of using it (with telegram account):
1. I have so many subscriptions, perhaps it would be useful to be able to alternate the list of all users with any folders created in telegram, this would make it much easier to keep an eye on important contacts.
2. perhaps a shortcut to follow the quoted messages is missing (or maybe I didn't find it)

I have tried nchat for a short time now, however, these are the only two things that I felt were necessary, for the rest, nice work!

**Comment by d99kris** [comment]
Hi @vinniec2 - regarding item 2, support for jumping to quoted message was added in v4.67, select the message and use `alt-q`.

Regarding folders, there is currently no plan to add support for it (also see #200).

**Comment by vinniec2** [comment]
For the shortcut, at that time I was consulting commands from the phone and I hadn't seen the list of commands under "interactive commands," sorry I didn't see it.
For the first point, ok, I'll try to find a way to use nchat anyway, I generally divide important people into separate folders so I don't miss their messages.
I am trying nchat on alpine in diskless mode and I must say it is much lighter than telegram web (500mb in one tab) and deskto client (sometimes consumes 800mb of ram), nchat runs around 100-150mb without having to download additional dependencies, so definitely a better choice.

:)
